backend/src/server.ts

```diff
@@ -109,14 +109,22 @@ app.post("/api/groups", async (req, res) => {
   const accessToken = req.headers.access_token;
   const user = await getUser(accessToken);
 
-  const created = createGroup(
+  const createdGroup = createGroup(
     user.id,
     subscription.name,
     subscription.cost,
     new Date(),
     subscription.image
   );
-
+  for (const memberData of friends) {
+    const createdFriend = await createMember(
+    
```

Comment:
just missing the group ID to attach it to the group

Reply:
not sure if this is the way to do it.

Reply:
Looks like it'll work! Did you test it?

Reply:
When I run the backend I encounter a network error some kind of authentication error from supabase this was before I did any changes. Check discord.